models/src/main/scala/it/pagopa/interop/authorizationmanagement/model/client/PersistentClientPurpose.scala

```diff
@@ -1,5 +1,3 @@
 package it.pagopa.interop.authorizationmanagement.model.client
 
-import java.util.UUID
-
-final case class PersistentClientPurpose(id: UUID, statesChain: PersistentClientStatesChain)
+final case class PersistentClientPurpose(statesChain: PersistentClientStatesChain)
```

Comment:
I left this intermediate level to be able to easily add something in the future, but let me know if you disagree